File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/recipedto/d/RecipeDTO.java
package iih.ci.ord.dto.recipedto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 处方信息 DTO数据 
 * 
 */
public class RecipeDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 药品处方主键
	 * @return String
	 */
	public String getId_pres() {
		return ((String) getAttrVal("Id_pres"));
	}
	/**
	 * 药品处方主键
	 * @param Id_pres
	 */
	public void setId_pres(String Id_pres) {
		setAttrVal("Id_pres", Id_pres);
	}
	/**
	 * 诊断序号
	 * @return String
	 */
	public String getSortno() {
		return ((String) getAttrVal("Sortno"));
	}
	/**
	 * 诊断序号
	 * @param Sortno
	 */
	public void setSortno(String Sortno) {
		setAttrVal("Sortno", Sortno);
	}
	/**
	 * 诊断编码
	 * @return String
	 */
	public String getDidef_code() {
		return ((String) getAttrVal("Didef_code"));
	}
	/**
	 * 诊断编码
	 * @param Didef_code
	 */
	public void setDidef_code(String Didef_code) {
		setAttrVal("Didef_code", Didef_code);
	}
	/**
	 * 诊断名称
	 * @return String
	 */
	public String getDidef_name() {
		return ((String) getAttrVal("Didef_name"));
	}
	/**
	 * 诊断名称
	 * @param Didef_name
	 */
	public void setDidef_name(String Didef_name) {
		setAttrVal("Didef_name", Didef_name);
	}
	/**
	 * 病历信息
	 * @return String
	 */
	public String getMr_content() {
		return ((String) getAttrVal("Mr_content"));
	}
	/**
	 * 病历信息
	 * @param Mr_content
	 */
	public void setMr_content(String Mr_content) {
		setAttrVal("Mr_content", Mr_content);
	}
	/**
	 * 开立科室
	 * @return String
	 */
	public String getId_dep_or() {
		return ((String) getAttrVal("Id_dep_or"));
	}
	/**
	 * 开立科室
	 * @param Id_dep_or
	 */
	public void setId_dep_or(String Id_dep_or) {
		setAttrVal("Id_dep_or", Id_dep_or);
	}
	/**
	 * his科室编码
	 * @return String
	 */
	public String getCode_dep() {
		return ((String) getAttrVal("Code_dep"));
	}
	/**
	 * his科室编码
	 * @param Code_dep
	 */
	public void setCode_dep(String Code_dep) {
		setAttrVal("Code_dep", Code_dep);
	}
	/**
	 * 医保计划下科别编码
	 * @return String
	 */
	public String getDepcode_hp() {
		return ((String) getAttrVal("Depcode_hp"));
	}
	/**
	 * 医保计划下科别编码
	 * @param Depcode_hp
	 */
	public void setDepcode_hp(String Depcode_hp) {
		setAttrVal("Depcode_hp", Depcode_hp);
	}
	/**
	 * 医保计划下科别名称
	 * @return String
	 */
	public String getDepname_hp() {
		return ((String) getAttrVal("Depname_hp"));
	}
	/**
	 * 医保计划下科别名称
	 * @param Depname_hp
	 */
	public void setDepname_hp(String Depname_hp) {
		setAttrVal("Depname_hp", Depname_hp);
	}
	/**
	 * 开立医生
	 * @return String
	 */
	public String getId_emp_or() {
		return ((String) getAttrVal("Id_emp_or"));
	}
	/**
	 * 开立医生
	 * @param Id_emp_or
	 */
	public void setId_emp_or(String Id_emp_or) {
		setAttrVal("Id_emp_or", Id_emp_or);
	}
	/**
	 * 开立科室名称
	 * @return String
	 */
	public String getId_dep_name() {
		return ((String) getAttrVal("Id_dep_name"));
	}
	/**
	 * 开立科室名称
	 * @param Id_dep_name
	 */
	public void setId_dep_name(String Id_dep_name) {
		setAttrVal("Id_dep_name", Id_dep_name);
	}
	/**
	 * 开立医生名称
	 * @return String
	 */
	public String getId_emp_name() {
		return ((String) getAttrVal("Id_emp_name"));
	}
	/**
	 * 开立医生名称
	 * @param Id_emp_name
	 */
	public void setId_emp_name(String Id_emp_name) {
		setAttrVal("Id_emp_name", Id_emp_name);
	}
	/**
	 * 处方类型
	 * @return String
	 */
	public String getId_prestp() {
		return ((String) getAttrVal("Id_prestp"));
	}
	/**
	 * 处方类型
	 * @param Id_prestp
	 */
	public void setId_prestp(String Id_prestp) {
		setAttrVal("Id_prestp", Id_prestp);
	}
	/**
	 * 处方编码
	 * @return String
	 */
	public String getSd_prestp() {
		return ((String) getAttrVal("Sd_prestp"));
	}
	/**
	 * 处方编码
	 * @param Sd_prestp
	 */
	public void setSd_prestp(String Sd_prestp) {
		setAttrVal("Sd_prestp", Sd_prestp);
	}
	/**
	 * 代开药标志
	 * @return FBoolean
	 */
	public FBoolean getHelpmedicineflag() {
		return ((FBoolean) getAttrVal("Helpmedicineflag"));
	}
	/**
	 * 代开药标志
	 * @param Helpmedicineflag
	 */
	public void setHelpmedicineflag(FBoolean Helpmedicineflag) {
		setAttrVal("Helpmedicineflag", Helpmedicineflag);
	}
	/**
	 * 处方时间
	 * @return FDateTime
	 */
	public FDateTime getDt_entry() {
		return ((FDateTime) getAttrVal("Dt_entry"));
	}
	/**
	 * 处方时间
	 * @param Dt_entry
	 */
	public void setDt_entry(FDateTime Dt_entry) {
		setAttrVal("Dt_entry", Dt_entry);
	}
	/**
	 * 备注
	 * @return String
	 */
	public String getDes() {
		return ((String) getAttrVal("Des"));
	}
	/**
	 * 备注
	 * @param Des
	 */
	public void setDes(String Des) {
		setAttrVal("Des", Des);
	}
	/**
	 * 挂号交易流水号
	 * @return String
	 */
	public String getRegistertradeno() {
		return ((String) getAttrVal("Registertradeno"));
	}
	/**
	 * 挂号交易流水号
	 * @param Registertradeno
	 */
	public void setRegistertradeno(String Registertradeno) {
		setAttrVal("Registertradeno", Registertradeno);
	}
	/**
	 * 单据类型
	 * @return String
	 */
	public String getBillstype() {
		return ((String) getAttrVal("Billstype"));
	}
	/**
	 * 单据类型
	 * @param Billstype
	 */
	public void setBillstype(String Billstype) {
		setAttrVal("Billstype", Billstype);
	}
	/**
	 * 本院科别名称
	 * @return String
	 */
	public String getHospital_dept_name() {
		return ((String) getAttrVal("Hospital_dept_name"));
	}
	/**
	 * 本院科别名称
	 * @param Hospital_dept_name
	 */
	public void setHospital_dept_name(String Hospital_dept_name) {
		setAttrVal("Hospital_dept_name", Hospital_dept_name);
	}
	/**
	 * 患者主键
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者主键
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 就诊号
	 * @return String
	 */
	public String getId_en() {
		return ((String) getAttrVal("Id_en"));
	}
	/**
	 * 就诊号
	 * @param Id_en
	 */
	public void setId_en(String Id_en) {
		setAttrVal("Id_en", Id_en);
	}
	/**
	 * 处方类型名称
	 * @return String
	 */
	public String getPrestp_name() {
		return ((String) getAttrVal("Prestp_name"));
	}
	/**
	 * 处方类型名称
	 * @param Prestp_name
	 */
	public void setPrestp_name(String Prestp_name) {
		setAttrVal("Prestp_name", Prestp_name);
	}
	/**
	 * 处方类型（1医保内，2 医保外）
	 * @return FBoolean
	 */
	public FBoolean getRecipetype() {
		return ((FBoolean) getAttrVal("Recipetype"));
	}
	/**
	 * 处方类型（1医保内，2 医保外）
	 * @param Recipetype
	 */
	public void setRecipetype(FBoolean Recipetype) {
		setAttrVal("Recipetype", Recipetype);
	}
	/**
	 * （费用使用）备注
	 * @return String
	 */
	public String getRemark() {
		return ((String) getAttrVal("Remark"));
	}
	/**
	 * （费用使用）备注
	 * @param Remark
	 */
	public void setRemark(String Remark) {
		setAttrVal("Remark", Remark);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/recipedto/d/RecipeDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.recipedto.d
{
    /// <summary>
    /// RecipeDTO 的摘要说明。
    /// </summary>
    public class RecipeDTO : BaseDTO {

        public RecipeDTO() {
 
        }

        /// <summary>
        /// 药品处方主键
        /// </summary>
		public string Id_pres {
            get { return getAttrVal<string>("Id_pres",null); }
            set { setAttrVal<string>("Id_pres", value); }
        }

        /// <summary>
        /// 诊断序号
        /// </summary>
		public string Sortno {
            get { return getAttrVal<string>("Sortno",null); }
            set { setAttrVal<string>("Sortno", value); }
        }

        /// <summary>
        /// 诊断编码
        /// </summary>
		public string Didef_code {
            get { return getAttrVal<string>("Didef_code",null); }
            set { setAttrVal<string>("Didef_code", value); }
        }

        /// <summary>
        /// 诊断名称
        /// </summary>
		public string Didef_name {
            get { return getAttrVal<string>("Didef_name",null); }
            set { setAttrVal<string>("Didef_name", value); }
        }

        /// <summary>
        /// 病历信息
        /// </summary>
		public string Mr_content {
            get { return getAttrVal<string>("Mr_content",null); }
            set { setAttrVal<string>("Mr_content", value); }
        }

        /// <summary>
        /// 开立科室
        /// </summary>
		public string Id_dep_or {
            get { return getAttrVal<string>("Id_dep_or",null); }
            set { setAttrVal<string>("Id_dep_or", value); }
        }

        /// <summary>
        /// his科室编码
        /// </summary>
		public string Code_dep {
            get { return getAttrVal<string>("Code_dep",null); }
            set { setAttrVal<string>("Code_dep", value); }
        }

        /// <summary>
        /// 医保计划下科别编码
        /// </summary>
		public string Depcode_hp {
            get { return getAttrVal<string>("Depcode_hp",null); }
            set { setAttrVal<string>("Depcode_hp", value); }
        }

        /// <summary>
        /// 医保计划下科别名称
        /// </summary>
		public string Depname_hp {
            get { return getAttrVal<string>("Depname_hp",null); }
            set { setAttrVal<string>("Depname_hp", value); }
        }

        /// <summary>
        /// 开立医生
        /// </summary>
		public string Id_emp_or {
            get { return getAttrVal<string>("Id_emp_or",null); }
            set { setAttrVal<string>("Id_emp_or", value); }
        }

        /// <summary>
        /// 开立科室名称
        /// </summary>
		public string Id_dep_name {
            get { return getAttrVal<string>("Id_dep_name",null); }
            set { setAttrVal<string>("Id_dep_name", value); }
        }

        /// <summary>
        /// 开立医生名称
        /// </summary>
		public string Id_emp_name {
            get { return getAttrVal<string>("Id_emp_name",null); }
            set { setAttrVal<string>("Id_emp_name", value); }
        }

        /// <summary>
        /// 处方类型
        /// </summary>
		public string Id_prestp {
            get { return getAttrVal<string>("Id_prestp",null); }
            set { setAttrVal<string>("Id_prestp", value); }
        }

        /// <summary>
        /// 处方编码
        /// </summary>
		public string Sd_prestp {
            get { return getAttrVal<string>("Sd_prestp",null); }
            set { setAttrVal<string>("Sd_prestp", value); }
        }

        /// <summary>
        /// 代开药标志
        /// </summary>
		public bool? Helpmedicineflag {
            get { return getAttrVal<FBoolean>("Helpmedicineflag",null); }
            set { setAttrVal<FBoolean>("Helpmedicineflag", value); }
        }

        /// <summary>
        /// 处方时间
        /// </summary>
		public DateTime? Dt_entry {
            get { return getAttrVal<FDateTime>("Dt_entry",null); }
            set { setAttrVal<FDateTime>("Dt_entry", value); }
        }

        /// <summary>
        /// 备注
        /// </summary>
		public string Des {
            get { return getAttrVal<string>("Des",null); }
            set { setAttrVal<string>("Des", value); }
        }

        /// <summary>
        /// 挂号交易流水号
        /// </summary>
		public string Registertradeno {
            get { return getAttrVal<string>("Registertradeno",null); }
            set { setAttrVal<string>("Registertradeno", value); }
        }

        /// <summary>
        /// 单据类型
        /// </summary>
		public string Billstype {
            get { return getAttrVal<string>("Billstype",null); }
            set { setAttrVal<string>("Billstype", value); }
        }

        /// <summary>
        /// 本院科别名称
        /// </summary>
		public string Hospital_dept_name {
            get { return getAttrVal<string>("Hospital_dept_name",null); }
            set { setAttrVal<string>("Hospital_dept_name", value); }
        }

        /// <summary>
        /// 患者主键
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 就诊号
        /// </summary>
		public string Id_en {
            get { return getAttrVal<string>("Id_en",null); }
            set { setAttrVal<string>("Id_en", value); }
        }

        /// <summary>
        /// 处方类型名称
        /// </summary>
		public string Prestp_name {
            get { return getAttrVal<string>("Prestp_name",null); }
            set { setAttrVal<string>("Prestp_name", value); }
        }

        /// <summary>
        /// 处方类型（1医保内，2 医保外）
        /// </summary>
		public bool? Recipetype {
            get { return getAttrVal<FBoolean>("Recipetype",null); }
            set { setAttrVal<FBoolean>("Recipetype", value); }
        }

        /// <summary>
        /// （费用使用）备注
        /// </summary>
		public string Remark {
            get { return getAttrVal<string>("Remark",null); }
            set { setAttrVal<string>("Remark", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_pres", "Sortno", "Didef_code", "Didef_name", "Mr_content", "Id_dep_or", "Code_dep", "Depcode_hp", "Depname_hp", "Id_emp_or", "Id_dep_name", "Id_emp_name", "Id_prestp", "Sd_prestp", "Helpmedicineflag", "Dt_entry", "Des", "Registertradeno", "Billstype", "Hospital_dept_name", "Id_pat", "Id_en", "Prestp_name", "Recipetype", "Remark"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.recipedto.d.RecipeDTO";
        }
    }
}
